### PR TITLE
Add --version option

### PR DESCRIPTION
**Pull request by san650**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -30,6 +30,7 @@ of the branches in the list.
 | `-m` `--modified`       | Show last modified branches |
 | `-i` `--no-interactive` | Don't use interactive mode. Interactive by default. |
 | `-c` `--count <number>` | Show number of branches. By default it shows nine.|
+| `--version`             | Show version number and quit |
```
Comment (eldano) [inline README.md]: For consistency, don't you think we should add `-v` ?

Comment (san650) [inline README.md]: I think it's a matter of taste, for example `git` doesn't have a `-v` option, only a `--version` option.

If you think it's important I can add it.

**Comment by san650** [comment]
@eldano https://github.com/san650/homebrew-git-switch/pull/1/files#diff-34cd4a6161b5db135c901ab96e39fda2R20

File: README.md (hunk 1 above)
Comment (eldano) [inline README.md]: The short option already works, OptParse does that for you automatically, so it seemed that it was missing in the documentation.

**Comment by san650** [comment]
@eldano I've update the documentation to include `-v`

**Pull request merged**